src/Nito.Mvvm.Async/CancelCommand.cs

```diff
@@ -118,14 +118,7 @@ public Func<Task> WrapCancel(Func<CancellationToken, Task> executeAsync)
             {
                 using (StartOperation())
                 {
-                    try
-                    {
-                        await executeAsync(parameter, _context.Token);
-                    }
-                    catch (OperationCanceledException)
-                    {
-                        // We cannot use `when (ex.CancellationToken == cancellationToken)` on the catch block because the user delegate may be cancelled by a linked CancellationToken.
-                    }
+                    await executeAsync(parameter, _context.Token);
```

Comment:
It looks like you accidentally included this code change in the wrong PR. This looks like a copy of the changes in https://github.com/StephenCleary/Mvvm/pull/31, which are not relevant here